### ISSUE TITLE
Possible conflict with the HTML Accessibility API Mappings document

### ISSUE THREAD

**Issue by matatk** [w3c/html-aria#63]
I have a question about an apparent conflict (or lack of info expressed in this document) relative to the [HTML Accessibility API Mappings 1.0](https://www.w3.org/TR/html-aam-1.0) document.

Just as an example, I am looking at the implicit semantics of the `<footer>` element.

In the table in [section 2 of this document](http://w3c.github.io/html-aria/#docconformance) the entry for `<footer>` states

> If not a descendant of an article, aside, main, nav - (changed) or section element

However, in the [HTML Accessibility API Mappings document's "HTML Element Role Mappings" section](https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings), other sectioning/sectioning root elements are added into the implicit mapping's description:

> descendant of body but not of main or other sectioning content or sectioning root element

Sectioning elements are as above, but sectioning root elements include `<blockquote>`, `<td>` and others. 

To me, it seems that the info provided in this document is thus not complete, as not all the nesting constraints are given; wonder if it would be best to refer to the HTML Accessibility API Mappings table as the "single point of truth" for this?

**Comment by asurkov** [comment]
there's a related issue 77 (https://github.com/w3c/html-aam/issues/77), where suggested to rely on HTML spec definition, which should resolve this issue, @jasonkiss ?

**Comment by jasonkiss** [comment]
Agreed. Would recommend reusing language from the [HTML-AAM](https://w3c.github.io/html-aam/) and the phrasing pattern already used in this spec for the section element:
- `role=banner` if the `header` element is scoped to the `body`. Otherwise, no corresponding role.
- `role=contentinfo` if the `footer` element is scoped to the `body`. Otherwise, no corresponding role.

**Comment by stevefaulkner** [comment]
the issue with these definitions is that it is not immediately clear what "scoped to the body" means

**Comment by asurkov** [comment]
agreed with @stevefaulkner we should have a link to the HTML spec. I'm curious if can refer to specific paragraph, rather than to element itself (http://w3c.github.io/html/sections.html#the-header-element)

**Comment by scottaohara** [comment]
Pulling in the comment from @JAWS-test from issue #161, as I'm closing that issue to consolidate:

>HTML-ARIA: "If not a descendant of an article, aside, main, nav - (changed) or   section element"
HTML AAM: "scoped to the main element, a sectioning content element, or a sectioning root element other than body"  

>Definition of sectioning root: https://html.spec.whatwg.org/multipage/sections.html#sectioning-root  

>I don't care which of the specifications is adapted, but it should be uniform. See also: https://github.com/w3c/html-aam/issues/222

**Comment by scottaohara** [comment]
Comment I had originally posted in #118, but going to post/revise here because I think we should merge 118 and then update again:

---

What's missing now is mention of the [sectioning root elements (sans body)](https://html.spec.whatwg.org/multipage/sections.html#sectioning-root), which if those elements were added individually, it'd require listing:     `blockquote` `details`, `dialog`, `fieldset`, `figure`, `td` and mirroring applicable ARIA roles.

Might this be something we instead reduce to just mentioning the high level requirement of "if scoped to the body [banner/contentinfo], if else no corresponding role" and then link "scoped to the body" to an in-document reference of what that means, including the types of elements and ARIA roles that would prevent a header/footer from being scoped to the body?   

Should we also consider noting other ARIA roles that should result in a header/footer not being mapped to landmarks?  Quickly reviewing, I'm personally thinking that most of ARIA's roles would also prevent these elements from becoming landmarks.... to the point where it'd likely be fewer roles to list to say what would allow header/footer to be landmarks than to list the ones that would not allow such mappings.

**Comment by scottaohara** [comment]
resolving this issue needs https://github.com/w3c/html-aam/issues/375 to be completed.
there is no longer the concept of a sectioning root in HTML with the recent update to the outline algorithm in the spec.
